tests/mistake_scanner/mistake_scanner.sh

```diff
@@ -0,0 +1,15 @@
+# Search for `return S2N_ERR_TYPE`. Should be `return S2N_ERROR(S2N_ERR_TYPE);`
```

Comment:
Needs copyright info. You can copy and paste it from https://github.com/awslabs/s2n/blob/master/.travis/run_cppcheck.sh